docker-compose.yml

```diff
@@ -1,6 +1,18 @@
 version: '3.6'
-services:
+x-rust-service:
+  &rust-service
```

Comment:
Is this some sort of tag? referencing the `*rust-service`s appearing later?

Reply:
Check out https://docs.docker.com/compose/compose-file/#extension-fields